src/devices_models/devices/hydro_generation.jl

```diff
@@ -330,12 +349,12 @@ function _get_budget(psi_container::PSIContainer,
         tech = PSY.get_tech(device)
         # This is where you would get the water/energy storage capacity
         # which is then multiplied by the forecast value to get you the energy budget
-        energy_capacity = use_forecast_data ? PSY.get_storagecapacity(device) : PSY.get_activepower(device)
+        energy_capacity = use_forecast_data ? PSY.get_storage_capacity(device) : PSY.get_activepower(device)
```

Comment:
we need to eliminate ternary operators.

Reply:
IMHO, that was a completely acceptable use of ternary operators.  Discussed here:  https://github.com/NREL/PowerSystems.jl/blob/master/docs/src/developer/style.md#one-line-conditionals